### ISSUE TITLE
Readme example doesn't work

### ISSUE THREAD

**Issue by AlexD97** [tecosaur/Org.jl#10]
The readme example does not work.
`text2 = parse(Org, "Some *Org* markup, written with easy using the ~parse~ function.")`
gives

```
ERROR: MethodError: no method matching parse(::Module, ::String)
Closest candidates are:
  parse(::Type{T}, ::AbstractString) where T<:Complex at /usr/share/julia/base/parse.jl:381
  parse(::Type{Sockets.IPAddr}, ::AbstractString) at /usr/share/julia/stdlib/v1.7/Sockets/src/IPAddr.jl:246
  parse(::Type{T}, ::AbstractString; base) where T<:Integer at /usr/share/julia/base/parse.jl:240
  ...
Stacktrace:
 [1] top-level scope
   @ REPL[4]:1

```

**Comment by tecosaur** [comment]
Ah yes, that needs updating. Try `OrgDoc` instead.

**Comment by tecosaur** [comment]
Fixed in 3890337766e9c1734fc0941275b16171ced49f7f
